assets/app/views/directives/environment.html

```diff
@@ -3,10 +3,11 @@
     <tbody>
       <tr ng-repeat="env in envVars">
         <td>{{env.name}}</td>
-        <td>
+        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
+        <td style="word-break: break-all;">
```

Comment:
ok nevermind... there were two IEs in the w3schools list :)  they had edge
listed separately from the old IEs. so yes it does seem to be supported

On Tue, Mar 8, 2016 at 4:11 PM, Jessica Forrester <jforrest@redhat.com>
wrote:

> although the doc you linked says something different than what i was
> looking at.  so now im not sure
>
> On Tue, Mar 8, 2016 at 4:10 PM, Jessica Forrester <jforrest@redhat.com>
> wrote:
>
>> bah sorry said the wrong thing, word-break is not supported
>>
>> On Tue, Mar 8, 2016 at 4:09 PM, Sam Padgett <notifications@github.com>
>> wrote:
>>
>>> In assets/app/views/directives/environment.html
>>> <https://github.com/openshift/origin/pull/7868#discussion_r55427719>:
>>>
>>> > @@ -3,10 +3,11 @@
>>> >      <tbody>
>>> >        <tr ng-repeat="env in envVars">
>>> >          <td>{{env.name}}</td>
>>> > -        <td>
>>> > +        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
>>> > +        <td style="word-break: break-all;">
>>>
>>> Is our mixin using the wrong property? I though it was word-break:
>>> break-all and word-wrap: break-word... Different properties.
>>>
>>>
>>> https://github.com/openshift/origin/blob/master/assets/app/styles/_mixins.less#L42-L47
>>> https://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap
>>> https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/openshift/origin/pull/7868/files#r55427719>.
>>>
>>
>>
>